Texstyle/Texstyle/Text.swift

```diff
@@ -153,13 +153,24 @@ public final class Text {
         }
         return attributedString.boundingRect(with: size, options: options, context: context)
     }
+
+    public func copy() -> Text {
+        let stylesCopy = styles.mapValues { style in
+            style.copy()
+        }
```

Comment:
Is it possible to do just `let stylesCopy = styles.copy()`?

Reply:
No, especially because of `mapValues` function